### ISSUE TITLE
Radix connect button should be accessible

### ISSUE THREAD

**Issue by attilacsanyi** [radixdlt/connect-button#57]
After adding `radix-connect-button` web component and running **Lighthouse Accessibility** report it shows that the button has not specified accessible name:

<img src="https://github.com/radixdlt/connect-button/assets/2144794/c2b2e98e-3716-4be5-8093-bd255e55a5df" width="600px" />

Should be an easy win adding something like `aria-label="Radix Connect Button"` [here](https://github.com/radixdlt/connect-button/blob/main/src/components/button/button.ts#L101) on the `button`.
